pages/common/ia.md

```diff
@@ -0,0 +1,24 @@
+# ia
+
+> Command-line tool to interact with archive.org.
+> More information: <https://archive.org/services/docs/api/internetarchive/cli.html>.
+
+- Configure ia to get your API keys; certain functions won't work without this step:
+
+`ia configure`
+
+- Upload item(s) to archive.org:
+
+`ia upload {{identifier_value}} {{file_name}} --metadata="{{mediatype_value}}" --metadata="{{metadata_value}}"`
```

Comment:
Perhaps this would be more consistent with other existing pages?
```suggestion
`ia upload {{identifier_value}} {{path/to/file}} --metadata="{{mediatype_value}}" --metadata="{{metadata_value}}"`
```